### ISSUE TITLE
Usage of NatGradModel/Wrapper to train variational parameters

### ISSUE THREAD

**Issue by izsahara** [secondmind-labs/GPflux#35]
Is there a minimum example for training the variational parameters with the Natural Gradient optimizer of `gpflux.optimization.keras_natgrad` for a deepgp model?
